src/codegen/impl_debug.rs

```diff
@@ -184,28 +186,40 @@ impl<'a> ImplDebug<'a> for Item {
                     // The simple case
                     debug_print(name, quote! { #name_ident })
                 } else {
-                    // Let's implement our own print function
+                    if ctx.options().use_core {
```

Comment:
Maybe point out that there's no `String` in `core`.

Reply:
Done.